main.go

```diff
@@ -53,3 +71,31 @@ func main() {
 
 	srv.Stop()
 }
+
+func defaultLoggingConfig() string {
+	return `
+	{
+		"def": {
+			"type": "console",
+			"options": {
+				"out": "stdout"
```

Comment:
Is there any way to infer this out of the old config if no config is passed? I want to keep to the minimum the hassle for people upgrading.